test/prototype/test_low_bit_optim.py

```diff
@@ -150,6 +150,57 @@ def test_optim_smoke(self, optim_name, dtype, device):
 
         for p1, p2 in zip(model.parameters(), model2.parameters()):
             torch.testing.assert_close(p2, p1)
+    
+    @pytest.mark.skipif(
+        not TORCH_VERSION_AT_LEAST_2_4, reason="FP8 CUDA requires PyTorch >= 2.4"
+    )
+    @pytest.mark.skipif(
+        torch.cuda.get_device_capability() < (8, 9), reason="FP8 CUDA requires compute capability >= 8.9"
+    )
+    @parametrize(
+        "optim_name",
+        ["AdamFp8", "AdamWFp8"],
+    )
+    @parametrize("dtype", [torch.float32, torch.bfloat16])
+    @parametrize("device", _DEVICES)
+    @parametrize("optim_addon",["dynamic_range_expansion"])
+    def test_optim_addons(self, optim_name, dtype, device, optim_addon):
```

Comment:
You can make this `test_optim_fp8_coat_smoke()`. Don't need `optim_addon`, since when `dynamic_range_expansion=False`, we already tested it in the generic smoke test.